### DISCUSSION TITLE
Error: BRouter Download Window

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#350] [Q&A] [answered]
Since the last beta, sometimes the BRouter Download Window occurs when I am tapping on the map. It is not regularly. Sometimes when I get the app from background to foreground. I do not use the BRouter. Can you have a look to this strange behaviour? 
![IMG20231022190737](https://github.com/devemux86/cruiser/assets/138766725/a2a0bd13-cc6d-419a-9af0-71d26ca4953a)

**Comment by kds112** [comment]
Screenshot did not work, therefore a photo from the screen....

**Comment by devemux86** [comment, marked as answer]
This message appears when we select BRouter (offline) as routing service and BRouter app is not installed.

What routing service do you have selected?
If you do not use BRouter, you can leave it to default OpenRouteService or something else?

**Reply by kds112** [reply]
Ahhh... Okay! It was set to BRouter, I changed it to OpenRoute and everything is ok. Thanks for the quick help 👍
